### PR TITLE
BD-786 : Request To Scan Metrological Github meta-wpe

### PR DESCRIPTION
**Pull request by ansu-m**

### CODE REVIEW

**Comment by wouterlucas** [comment]
Are we doing this to just morty? What about krogoth, master and other Yocto releases?

Also this:
```
Before RDK accepts your code into the project you must sign the RDK Contributor License Agreement (CLA).
```

IMHO Does not belong here. This layer is part of Yocto and would be misaligned with what Yocto/OpenEmbedded stands for, it is not exclusive to RDK.

You missed some details:
```
   Copyright <YEAR> <COPYRIGHT HOLDER>
```

Needs year and holder.

File: tools/check_srcrevs.py [hunk 1]
```diff
@@ -10,18 +27,18 @@ def findRev(file):
...
+    fd = open(file, 'r')
+    file_content = fd.read()
+    fd.close()
 
     # Use RE package to allow for replacement (also allowing for (multiline) REGEX)
-    file_string = (re.sub("[a-z0-9]{40}", newVersion, file_string))
+    file_content = (re.sub("[a-z0-9]{40}", newVersion, file_content))
 
     # Write contents to file.
     # Using mode 'w' truncates the file.
```
Comment (mhughesacn) [inline tools/check_srcrevs.py]: Change the comments at 29, 34 and 37 so they are not identical and therefore a direct copy.  e.g.
30: # Get file contents, multiple lines returned as one string
34: # Re package will handle multiline input string in substitution
37/38: # Write contents and truncate file.

**Comment by mhughesacn** [comment]
Hi Ansu, Please fix the MIT license as Wouter requested as well.

**Comment by woutermeek** [comment]
I don't believe this PR is needed anymore, we can re-open if we decide it is needed later

**Pull request closed without merging**